packages/element-lib/test/Truffle-Tests/Element Contract Test Explainer.md

```diff
@@ -0,0 +1,90 @@
+# Element Contract Test Explainer
```

Comment:
This is one of the. nicest PRs, thanks for adding so much documentation!

Reply:
Thank you @OR13 -- too kind